bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -4507,21 +4507,21 @@ spec:
       name: metrics-powermax
     - image: quay.io/dell/container-storage-modules/csm-metrics-powerflex:v1.11.0
       name: metrics-powerflex
-    - image: quay.io/dell/container-storage-modules/podmon:v1.12.0
+    - image: quay.io/dell/container-storage-modules/podmon:v1.13.0
```

Comment:
why is podmon updated? it's not part of the helm chart PR we are replicating. Also, there are some instances of podmon that are still 1.12.